index.js

```diff
@@ -49,6 +51,15 @@ mb.app.on('quit', () => {
 	mb = null;
 });
 
+mb.on('ready', () => {
+	windowState = windowStateKeeper()
+	windowState.manage(mb.window);
+})
+
+mb.on('after-show', () => {
+	mb.window.setPosition(windowState.x, windowState.y, false);
```

Comment:
We need to check here if `x` and `y` properties are actual numbers because it's undefined by default.